doc/index.rst

```diff
@@ -43,7 +47,7 @@ ______
 API documentation
 =================
 
-Encoders
+Encoders / Vectorizers
 ------------
```

Comment:
I think that you need to expend the title line below

Reply:
Addressed in 01cf92c